Source/Scene/Cesium3DTile.js

```diff
@@ -735,6 +738,28 @@ define([
         return frameState.mode !== SceneMode.SCENE3D ? tile._contentBoundingVolume2D : tile._contentBoundingVolume;
     }
 
+
+    var scratchPlane = new Plane(Cartesian3.UNIT_X, 0.0);
+    function getCheckClippedFunction (tile) {
+        return function (boundingVolume) {
+            tile._debugClipped = false;
+
+            var planes = tile._tileset.clippingPlanes;
+            var length = planes.length;
+            for (var i = 0; i < length; ++i) {
+                var plane = planes[i];
+                Plane.transformPlane(plane, tile._tileset._root.computedTransform, scratchPlane);
+                if (boundingVolume.intersectPlane(scratchPlane) === Intersect.INSIDE) {
+                    tile._debugClipped = true;
```

Comment:
I'm not sure why this works... Is `Intersect.INSIDE` the result if the contents are completely on the opposite side of the clipping plane? Does this mean the normal is reverse what it should be?

Reply:
I wonder if the tileset should contain a culling volume built from its clipping planes. The check here should be functionally the same as `CullingVolume.computeVisibility`, so it would be better to reuse that code.

Reply:
> I'm not sure why this works... Is Intersect.INSIDE the result if the contents are completely on the opposite side of the clipping plane? Does this mean the normal is reverse what it should be?

I think I assumed we wanted to clip what was on the "inside" of the plane. If we want the "outside" to be clipped, it should be fairly easy to update the shader to clip the opposite of what we are now, and check for outside intersections instead.

Reply:
> I wonder if the tileset should contain a culling volume built from its clipping planes. The check here should be functionally the same as CullingVolume.computeVisibility, so it would be better to reuse that code.

I think the only difference is that we only care if the volume is inside (or outside after the above changes)

Reply:
Actually, another difference is that `CullingVolume` uses the `Cartesian4` representation for the coefficients of the plane equation to calculate intersections, rather than a `Plane` object with normal/distance. So I'm going to go with no clipping volume unless we want to convert the planes to their equations everytime we update.